src/microservices/Microservices.CohortExtractor/Messaging/ExtractionRequestQueueConsumer.cs

```diff
@@ -36,36 +34,40 @@ public class ExtractionRequestQueueConsumer : Consumer
         protected override void ProcessMessageImpl(IMessageHeader header, BasicDeliverEventArgs deliverArgs)
         {
             ExtractionRequestMessage request;
-
             if (!SafeDeserializeToMessage(header, deliverArgs, out request))
                 return;
 
-            Logger.Info("Received message for job " + request.ExtractionJobIdentifier);
+            Logger.Info($"Received message: {request}");
 
             _auditor.AuditExtractionRequest(request);
 
-            if (!CheckValidRequest(request, header, deliverArgs))
-                return;
+            if (!request.ExtractionDirectory.StartsWith(request.ProjectNumber))
```

Comment:
This seems quite specific, how deep is our dependency on project numbers and file system layouts?

Reply:
So the code was already there just moved around a bit.  Still don't like it very much.